universal-application-tool-0.0.1/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -35,7 +35,7 @@
 
   @Inject
   public ProgramApplicationListView(AdminLayout layout) {
-    this.layout = checkNotNull(layout).setProgramAdminType();
+    this.layout = checkNotNull(layout).setOnlyProgramAdminType();
```

Comment:
I think this logic made more sense when civiform admins couldn't also be program admins, since this page would only be visible to program admins so you could make this assumption. Was thinking that ideally we'd want to check what admin type this is and only call this if the user is only a program admin.

Happy to defer this to a future PR, since the ideal implementation might involve a bit of a refactor to consolidate the logic across ProgramApplicationListView, ProgramApplicationView, and perhaps others.